src/components/HomeClient/GameListCarousel.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+  GameListCard,
+} from '@/components';
+import { MODAL_TYPE } from '@/constants/modal';
+import useModalStore from '@/store/useModalStore';
+import { GamesResponse } from '@/types/api';
+
+interface GameListCarouselProps {
+  games: GamesResponse[];
+}
+
+const GameListCarousel = ({ games }: GameListCarouselProps) => {
+  const { openModal } = useModalStore();
+
+  const MAX_CAROUSEL_ITEMS = 6;
+  const isMultiplePages = games.length > MAX_CAROUSEL_ITEMS;
+
+  return (
+    <Carousel
+      opts={{
+        active: isMultiplePages ? true : false,
+      }}
```

Comment:
P4; 
이렇게 바꿔도 깔끔하지 않을까요? 바로 위에 정의된 사항이 있어서 찾기 어렵지 않을 것 같아서요!
```suggestion
    <Carousel opts={{ active: isMultiplePages }}>
```